src/object-properties.js

```diff
@@ -56,4 +68,49 @@ const properties = {
   },
 };
 
+/**
+ * Holds styling options
+ * @typedef {object} Style
+ * @property {FontColor=} fontColor - Color of the text, by default #484848
+ * @property {Background=} backgroundColor - Color of the background, by default #ffffff
+ * @property {Border=} border - Styling for border, by default #737373
+ */
+
+/**
+ * @typedef {object} FontColor
+ * @property {('auto'|'colorPicker'|'byExpression')=} [colorType='auto'] How the font color is defined
+ * @property {PaletteColor=} color - Color defined by index or hex code, needed if colorType is colorPicker
+ * @property {ColorExpression=} colorExpression - Color defined by expression, needed if colorType is byExpression
+ */
+
+/**
+ * @typedef {object} Background
+ * @property {('auto'|'colorPicker'|'byExpression')=} [colorType='colorPicker'] How the font color is defined
+ * @property {PaletteColor=} color - Color defined by index or hex code, needed if colorType is colorPicker
+ * @property {ColorExpression=} colorExpression - Color defined by expression, needed if colorType is byExpression
+ */
```

Comment:
These two are identical, could be one generic color type

Reply:
They are basically the same, but they have different default values for colorType

Reply:
you're right, my bad! It still says `font` on line 88 though